### ISSUE TITLE
REFPROP wrongly reports superheated phase below dew point for water + nitrogen + oxygen system

### ISSUE THREAD

**Issue by ghost** [usnistgov/REFPROP-issues#666]
![thumbnail_Outlook-gywzwaar](https://github.com/usnistgov/REFPROP-issues/assets/174985347/3a4fa4fb-6de7-4d0b-80bb-1acf1390a8a0)

### Description

For the water + oxygen + nitrogen system, the superheated phase is obtained both above and below dew point. 

### Steps to Reproduce

1. Select water, oxygen and nitrogen mixture in REFPROP
2. Enter the compositions Mass fraction 0.014/0.75633/0.22967
3. Enable "Quality" and "Phase" from Options->Properties 
4. In Calculate->Specific State Points, specify pressure as 103 kPa and Quality = 1.0
5. Do a PT flash just above and below the obtained dew point.
6. Observe the calculated Phase.

**Expected behavior:** Below dew point, the phase should be two phase.

**Actual behavior:** Below dew point, the phase is Superheated.

### Versions:

**REFPROP Version:** REFPROP 10.0
**Operating System and Version:** Windows 10 Enterprise  
**Access Method:** REFPROP software  

### Additional Information

Screenshot is attached.

**Comment by ianhbell** [comment]
What does the phase envelope look like in a P-T plot for this mixture? It is non type-I mixture and those are tricky in general and sometimes have problems in REFPROP.

**Comment by ghost** [comment]
The "Isoproperty Tables" shows that the PT flash fails for all temperatures less than dew point at 103.0 kPa. However, with the "Specified State Point", REFPROP calculates the results and wrongly shows "Superheated" phase. 

Can you please provide some resource where we can get more information on the Type-I and non-Type-I mixtures and how REFPROP handles it?

**Comment by ianhbell** [comment]
Can you please show the p-T plot I asked for previously?

A book you could read to understand the general problem of EOS typology is https://shop.elsevier.com/books/high-pressure-fluid-phase-equilibria/deiters/978-0-443-13280-3 .

**Comment by ghost** [comment]
Here is the PT diagram for the air+water mixture:
![PT diagram](https://github.com/user-attachments/assets/fb51c790-4540-4f93-9ddd-ea9c0d3890d2)

**Comment by ianhbell** [comment]
Do have saturation splines enabled in the options?

**Comment by ghost** [comment]
Yes, in this case, the "Initialize Saturation Boundaries for Mixture" is turned on. Surprisingly, redoing the PT calculations without changing the model options produced following plot:
![ptdiagram](https://github.com/user-attachments/assets/966a1d4a-0c62-4838-b145-fe31ef56cf74)

Turning off the spline produces following result:
![withoutSpline](https://github.com/user-attachments/assets/5e6fcd23-f7f8-435d-84cb-a1e5468f71d2)

**Comment by ianhbell** [comment]
This is a non-Type-I mixture so calculations are always going to be difficult for REFPROP. Aside from the use of the saturation splines, the only recommendation I can offer is to do the calculations with your own iterative solver.
